learn/update_and_migration/updating.mdx

```diff
@@ -36,7 +36,7 @@ The response should look something like this:
 }
 ```
 
-<Capsule intent="tip" title="If you get the `missing_authorization_header` error, you might be using **v0.24 or below**. For each command, replace the `Authorization: Bearer` header with the `X-Meili-API-Key: API_KEY` header:">
+<Capsule intent="tip" title={<>If you get the <code>missing_authorization_header</code> error, you might be using **v0.24 or below**. For each command, replace the <code>Authorization: Bearer</code> header with the <code>X-Meili-API-Key: API_KEY</code> header:</>}>
```

Comment:
Actually, does this even need to be a title? Could it just be part of the content?